ClientApp/viking-vault/src/display_user_list/DeleteUserModal.tsx

```diff
@@ -14,7 +14,7 @@ class DeleteUserModal extends React.Component<IDeleteUserModalProps, any>{
     }
 
     private closeModal = () =>{
```

Comment:
If this method's only job is to call a parent method received in props, please remove it. 
A props received parent method can be called from any event (e.g. `onClick`).